<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calendar</class>
 <widget class="QDialog" name="Calendar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>337</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>352</width>
    <height>176</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Qsalat - Calendar generation </string>
  </property>
  <widget class="QPushButton" name="generateButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate calendar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Generate salat times calendar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>381</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Select monthes</string>
   </property>
   <widget class="QCheckBox" name="checkBox_01">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>40</y>
      <width>94</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>January</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_04">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>94</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>April</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_07">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>94</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>July</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_02">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>94</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>February</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_03">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>94</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>March</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_05">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>94</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>May</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_06">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>94</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>June</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_08">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>94</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>August</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_09">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>September</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_11">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>94</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>November</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_12">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>94</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>December</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="unselectButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Unselect all</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>94</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>October</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
